README.md

```diff
@@ -1,5 +1,20 @@
 # Linter for Dart
 
+## [Repository has moved][]
+
+The Dart linter is now developed within the [Dart SDK][] repository,
+and can be found at https://github.com/dart-lang/sdk/tree/main/pkg/linter.
+
+Please open new linter issues on the [dart-lang/sdk repo][issues].
```

Comment:
Actually, let's hold off on the issue redirection for now?  We're not quite ready to migrate the issues and I think having to look in two places will be confusing.

Maybe we can call that out too? (That is, "File issues here for now..." or something similar.) WDYT?

Reply:
That's fine for me! 

Either way has a chance of issues placed on the other temporarily. Just consider adding a note to the issue moving _tracking issue_ to update this text when that is complete.

Reply:
This is great. Thanks!